code/modules/jobs/job_types/gimmick.dm

```diff
@@ -46,6 +47,7 @@
 	access = list(ACCESS_THEATRE, ACCESS_MAINT_TUNNELS)
 	minimal_access = list(ACCESS_THEATRE, ACCESS_MAINT_TUNNELS)
 	gimmick = TRUE
+	warning = "You play similarly to a clown or a mime, this job is not a license to grief."
```

Comment:
> You play similarly to a clown or a mime

This sounds like a license to grief.

Reply:
This was spurred by people claiming ignorance when getting banned from gimmicks

Clowns and mimes haven't been allowed to claim ignorance